src/register/Register.jsx

```diff
@@ -97,13 +98,14 @@ export default function Register() {
                             </p>
                             {!isValidName && (
                                 <span className="error">Must be between 2-30 characters long. Can only contain alphabets, whitespace and dash '-'</span>
-                            )}
+                             )}
                         </form-group>
                         <div><br></br></div>
                         <form-group controlId="formUserame">
```

Comment:
Can we please fix this typo here? `formUserName` from `formUserame` if this change don't require any change in other file.

Reply:
> Can we please fix this typo here? `formUserName` from `formUserame` if this change don't require any change in other files.

Fixed that, in my view, since the build is not failing it didn't require change in any other file.